src/NUnitConsole/nunit3-console/ConsoleRunner.cs

```diff
@@ -169,7 +169,14 @@ private int RunTests(TestPackage package, TestFilter filter)
             foreach (var spec in _options.ResultOutputSpecifications)
             {
                 var outputPath = Path.Combine(_workDirectory, spec.OutputPath);
-                GetResultWriter(spec).CheckWritability(outputPath);
+                try
+                {
+                    GetResultWriter(spec).CheckWritability(outputPath);
+                }
+                catch (UnauthorizedAccessException e)
```

Comment:
Better to catch any Exception here. I understand why you did this: there could be some unrelated exception in the extension, like a null reference. However extensions are supposed to have been tested separately, so at this point we have to trust the extension and assume the path is non-writable.